### ISSUE TITLE
Implements permissions.

### ISSUE THREAD

**Issue by ghost** [backdrop-contrib/custom_pub#3]
Last commit [implements permissions](https://github.com/backdrop-contrib/custom_pub/commit/e7d8db929dabd230223ec2f53f44228f51f664d4).

@klonos : if, of course, you agree with this implementation, would you be kind enough to find the words to describe it in the README.md ?

(that's the reason why I did not initially "translate" the original README.txt…)

**Comment by klonos** [comment]
Yes, I have already renamed the .txt to .md and will rework the text in it. In the morning though because :zzz: 

IIUC, this was the original README.txt from the d.org module as you have pulled it in the GitHub repo, right?
